### PR TITLE
[Docs] Add Inference Doc

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Add Inference Doc

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/user_guides/inference.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Inference with existing models
+
+MMClassification provides pre-trained models for classification in [Model Zoo](../model_zoo.md).
+This note will show **how to use existing models to inference on given images**.
+
+As for how to test existing models on standard datasets, please see this [guide](./train_test.md#Test)
+
+## Inference on given image
```
Comment (mzr1996) [inline docs/en/user_guides/inference.md:8]: a given image

File: docs/en/user_guides/inference.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+MMClassification provides high-level Python APIs for inference on given image:
```
Comment (mzr1996) [inline docs/en/user_guides/inference.md:10]: a given image

File: docs/en/user_guides/inference.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- [init_model](https://mmclassification.readthedocs.io/en/1.x/api/mmcls.apis.init_model.html#mmcls.apis.init_model): Initialize a model with a config and checkpoint
+- [inference_model](https://mmclassification.readthedocs.io/en/latest/api/mmcls.apis.init_model.html#mmcls.apis.inference_model): Inference on a given image
+
+Here is an example of building the model and inference on a given image by using ImageNet-1k pre-trained checkpoint.
+
+```{note}
+If you use mmcls as a 3rd-party package, you need to download the conifg and the demo image in the example.
```
Comment (mzr1996) [inline docs/en/user_guides/inference.md:18]: conifg -> config

**Review by mzr1996 — CHANGES_REQUESTED** [review]

File: docs/en/user_guides/inference.md (hunk 2 above)
Comment (Ezra-Yu) [reply docs/en/user_guides/inference.md]: done.

File: docs/zh_CN/user_guides/inference.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+至于如何在标准数据集上测试现有模型，请看这个[指南](./train_test.md#推理)
+
+## 对给定图像的推理
+
+MMClassification 为图像推理提供高级 Python API：
+
+- [init_model](mmcls.apis.init_model): 初始化一个模型。
+- [inference_model](mmcls.apis.inference_model)：对给定图片进行推理。
+
+下面是一个示例，如何使用一个 ImageNet-1k 预训练权重初始化模型并推里给定图像。
```
Comment (mzr1996) [inline docs/zh_CN/user_guides/inference.md:15]: 推里->推理

File: docs/zh_CN/user_guides/inference.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# 如何使用已有模型推理
```
Comment (mzr1996) [inline docs/zh_CN/user_guides/inference.md:1]:
```suggestion
# 使用现有模型推理
```

File: docs/zh_CN/user_guides/inference.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+MMClassification 在 [Model Zoo](../model_zoo.md) 中提供了用于分类的预训练模型。
+本说明将展示**如何使用现有模型对给定图像进行推理**。
+
+至于如何在标准数据集上测试现有模型，请看这个[指南](./train_test.md#推理)
+
+## 对给定图像的推理
```
Comment (mzr1996) [inline docs/zh_CN/user_guides/inference.md:8]:
```suggestion
## 推理单张图片
```

**Review by mzr1996 — CHANGES_REQUESTED** [review]

**Review by mzr1996 — APPROVED** [review]

**Pull request merged**